afm/filesystems/vault.py

```diff
@@ -15,53 +15,53 @@ def get_jwt_from_file(file_name):
 def vault_jwt_auth(jwt, vault_address, vault_path, role):
     """Authenticate against Vault using a JWT token (i.e., k8s sa token)"""
     full_auth_path = vault_address + vault_path
-    logging.debug("full_auth_path = %s", str(full_auth_path))
+    logger.debug("full_auth_path = %s", str(full_auth_path))
     json = {"jwt": jwt, "role": role}
     response = requests.post(full_auth_path, json=json)
     if response.status_code == 200:
         return response.json()
-    logging.critical("Got error code %d from Vault authentication", response.status_code)
-    logging.critical("Error response: %s", str(response.json()))
+    logger.critical("Got error code %d from Vault authentication", response.status_code)
```

Comment:
Separate error code from the message.  json log entry should be {"message" : "Got error from vault authentication", "error" : the error code value }

Reply:
Will address in future PR. Thanks